cms/envs/common.py

```diff
@@ -60,6 +60,9 @@
 
     # Django REST framework configuration
     REST_FRAMEWORK,
+
+    # Credentials service username
+    CREDENTIALS_SERVICE_USERNAME
```

Comment:
I suggest adding an inline comment about why this is being imported, e.g. "the following setting is referenced in a migration in openedx that gets run under both lms and cms, so it must be available in both places"